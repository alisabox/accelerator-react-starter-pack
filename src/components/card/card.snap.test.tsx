import { render } from '@testing-library/react';
import Card from './card';
import { guitars } from '../../mocks/mocks';

describe('Component: Card', () => {
  test('should render correctly vs. snapshot', () => {
    const { container } = render(<Card guitar={guitars[0]} />);
    expect(container).toMatchSnapshot();
  });
});
