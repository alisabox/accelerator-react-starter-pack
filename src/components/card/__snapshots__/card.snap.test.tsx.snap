// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Card should render correctly vs. snapshot 1`] = `
<div>
  <div
    class="product-card"
  >
    <img
      alt="Честер Bass"
      height="190"
      src="img/content/guitar-1.jpg"
      width="75"
    />
    <div
      class="product-card__info"
    >
      <div
        aria-hidden="true"
        class="rate product-card__rate"
      >
        <span
          class="visually-hidden"
        >
          Рейтинг:
        </span>
        <svg
          aria-hidden="true"
          height="11"
          width="12"
        >
          <use
            xlink:href="#icon-full-star"
          />
        </svg>
        <svg
          aria-hidden="true"
          height="11"
          width="12"
        >
          <use
            xlink:href="#icon-full-star"
          />
        </svg>
        <svg
          aria-hidden="true"
          height="11"
          width="12"
        >
          <use
            xlink:href="#icon-full-star"
          />
        </svg>
        <svg
          aria-hidden="true"
          height="11"
          width="12"
        >
          <use
            xlink:href="#icon-full-star"
          />
        </svg>
        <svg
          aria-hidden="true"
          height="11"
          width="12"
        >
          <use
            xlink:href="#icon-star"
          />
        </svg>
        <span
          class="rate__count"
        >
          0
        </span>
        <span
          class="rate__message"
        />
      </div>
      <p
        class="product-card__title"
      >
        Честер Bass
      </p>
      <p
        class="product-card__price"
      >
        <span
          class="visually-hidden"
        >
          Цена:
        </span>
        17500
         ₽
      </p>
    </div>
    <div
      class="product-card__buttons"
    >
      <a
        class="button button--mini"
        href="/#"
      >
        Подробнее
      </a>
      <a
        class="button button--red button--mini button--add-to-cart"
        href="/#"
      >
        Купить
      </a>
    </div>
  </div>
</div>
`;
